verification/curator-service/ui/src/components/SearchBar.tsx

```diff
@@ -192,22 +192,28 @@ export default function SearchBar(props: {
                 onClose={handleClose}
             >
                 {[
-                    'curator',
-                    'gender',
-                    'nationality',
-                    'occupation',
-                    'country',
-                    'outcome',
-                    'caseid',
-                    'sourceurl',
-                    'verificationstatus',
-                    'uploadid',
-                    'admin1',
-                    'admin2',
-                    'admin3',
-                ].map((text) => (
-                    <MenuItem key={text} onClick={(): void => clickItem(text)}>
-                        {text}
+                    { desc: 'curator email', value: 'curator' },
+                    { desc: 'gender', value: 'gender' },
+                    { desc: 'nationality', value: 'nationality' },
+                    { desc: 'occupation', value: 'occupation' },
+                    { desc: 'country', value: 'country' },
+                    { desc: 'outcome', value: 'outcome' },
+                    { desc: 'unique case ID', value: 'caseid' },
```

Comment:
nit: is 'unique' redundant when it's an ID? same for upload ID

Reply:
IDs are not necessarily "globally" unique, but yeah it's technical so not really relevant here, removed.